### DISCUSSION TITLE
A question about the PrxEncrypter thing

### DISCUSSION THREAD

**Discussion by Voxelstice** [pspdev/pspdev#58] [General]
What does it actually mean when the message ``PRX SIGNER: Elf is to big`` shows up when building a PSP game?
All it says is that the elf thing is just too big and that is it. No further info. i don't want to test the game with CFW.

i do not know if this is where i should be asking questions, i didnt want to join the discord server just for this only

**Comment by sharkwouter** [comment]
Which game are you trying to build? You don't have to encrypt, it can just be nice

**Comment by Voxelstice** [comment]
> Which game are you trying to build?

my own port of a existing game

> You don't have to encrypt

is it not really necessary sometimes?

**Comment by sharkwouter** [comment]
Encryption is only needed if want your program to run on official firmware. It would be helpful if you could share you code, though, then we can reproduce the issue.

**Comment by Voxelstice** [comment]
there's too much code to share (22 files), the game uses 10 libraries (7 being dependencies of some libraries, 3 being actually used)

**Comment by sharkwouter** [comment]
You can create a github repository for it or send the elf file.

**Comment by diamant3** [comment]
Hello guys, I will converting this now into discussion and feel free to continue!
